CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 - TODO ([#XXX](https://github.com/kedacore/keda/pull/XXX))
 - ScaledJob: introduce MultipleScalersCalculation ([#2016](https://github.com/kedacore/keda/pull/2016))
 - Add Graphite Scaler ([#1628](https://github.com/kedacore/keda/pull/2092))
+- Add Makefile mockgen targets ([#2090](https://github.com/kedacore/keda/issues/2090))
```

Comment:
Minor nit, could you please move this to `### Other` section, b/c it is not user facing issue. Thanks!

Reply:
Oops, done. I missed that part :)